StringSearch.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Scanner;
+import java.io.File;
+
+public class StringSearch {
+    private String s;
+    private String filename;
+
+    public StringSearch(String s, String filename){
```

Comment:
Maybe you don't need a constructor and instead send the arguments directly to search? Then you can use that method for many different searches without having to create a new instance of the class for each.